neurow/lib/neurow/jwt_auth_plug.ex

```diff
@@ -213,6 +219,19 @@ defmodule Neurow.JwtAuthPlug do
     end
   end
 
+  defp unauthorized(conn, error_code, error_message, options) do
+    Logger.error(
+      "JWT authentication error: #{error_code} - #{error_message}, path: '#{conn.request_path}'",
+      category: "security",
+      error_code: "jwt_authentication.#{error_code}",
+      authorization_header: conn |> get_req_header("authorization") |> List.first(),
```

Comment:
@papey  the `authorization_header` metadata is not automatically added to json logs. The header should be added to the error message also, and maybe `neurow/lib/neurow/ecs_log_formatter.ex` should be updated to add the authorization header in the JSON payload (while ensuring it conforms to the ECS log format)